### PR TITLE
Adding dotnet remove package sub command

### PR DESCRIPTION
**Pull request by mishra14**
Adding the `dotnet remove package` sub command to remove package references.

NuGet part of the work : https://github.com/NuGet/NuGet.Client/pull/1092

Fixes remove package part of #4521 and https://github.com/NuGet/Home/issues/4101

//cc: @livarcocc @piotrpMSFT @blackdwarf @krwq @rrelyea @emgarten

### CODE REVIEW

**Comment by mishra14** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build

File: src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Remove Package reference Command";
+
+        public const string AppDescription = "Command to remove package reference";
+
+        public const string AppHelpText = "Package references to remove";
+
+        public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to remove";
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs]: Should this say: Please specify only one package reference to remove? Also, add dots (.) to the sentences above.

@blackdwarf To review the text here.

Comment (blackdwarf) [inline src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs]: What @livarcocc said.

File: src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Remove.PackageReference
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Remove Package reference Command";
+
+        public const string AppDescription = "Command to remove package reference";
```
Comment (blackdwarf) [inline src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs]: `Command to remove a package reference.`

File: src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+        public const string AppHelpText = "Package references to remove";
```
Comment (blackdwarf) [inline src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs]: if we are only doing one, then this is singular, not plural.

**Comment by mishra14** [comment]
@livarcocc @blackdwarf : done.

**Comment by piotrpMSFT** [comment]
@blackdwarf can you review & approve?

**Comment by livarcocc** [comment]
@srivatsn

**Pull request merged**